hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/terasort/TeraGen.java

```diff
@@ -0,0 +1,301 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.example.hadoop.terasort;
+
+import io.pravega.example.hadoop.wordcount.PravegaOutputFormat;
+import io.pravega.example.hadoop.wordcount.TextSerializer;
```

Comment:
If `PravegaOutputFormat` and `TextSerializer` are common utilities, maybe they should be in another package, so any new hadoop example may easily use them.

Reply:
Yes there is a backlog item for it.
For the moment for the sake of terasort test, let's just leave the change here.